content/ecosystem/servers/servers.toml

```diff
@@ -18,6 +18,16 @@ licence = "Apache-2.0"
 repository = "https://github.com/matrix-construct/tuwunel"
 room = "#tuwunel:matrix.org"
 
+[[servers]]
+name = "Synapse Pro"
+description = "Synapse for Enterprise. Backs the Foundation's matrix.org homeserver."
```

Comment:
I'm not sure if this is the place and wording to advertise that fact. It would be very welcome though to share further info about the setup for example on https://matrix.org/homeserver/about/.

Also, in light of https://github.com/matrix-org/matrix.org/pull/2907#discussion_r2329757646 I would like @thibaultamartin's opinion here.

Reply:
https://github.com/matrix-org/matrix.org/pull/2122/files#r1444654419 Is another data point here for prior decisions on similar wording.

Reply:
I'm not super high temperature on adding the matrix.org use-case, though I thought it would be a useful fact to add since I often have to explain that matrix.org is backed by Synapse Pro rather than Synapse Community. 

I'll rewrite it instead and may follow up with details to the homeserver 'about' page. I like the idea of using the matrix.org homeserver as a success story case study.